app/javascript/components/Home.jsx

```diff
@@ -29,10 +29,13 @@ export default () => {
 
   // Fetch companies from API
   useEffect(() => {
+    const headers = new Headers();
+    headers.append("Authorization", "cb7615e298dd72441263f958e1306be1185fdff8");
```

Comment:
I want to clarify, this is not something to do in production. 
I didn't scaffold a user model with their own security tokens and a full jwt implementaiton.
This value could be fetched via an env variable using the dotenv npm package.

What I want to clarify is that I'm aware it's absolutely unsafe and a bad practice to have a token hardcoded in the code, and only did it as a draft.